externals/llvm-external-projects/torch-mlir-dialects/include/torch-mlir-dialects/Conversion/Passes.td

```diff
@@ -0,0 +1,25 @@
+#ifndef TORCH_MLIR_DIALECTS_CONVERSION_PASSES
+#define TORCH_MLIR_DIALECTS_CONVERSION_PASSES
+
+include "mlir/Pass/PassBase.td"
+
+#ifdef TORCH_MLIR_DIALECTS_ENABLE_TCP
+//===----------------------------------------------------------------------===//
+// TcpToTosa
+//===----------------------------------------------------------------------===//
+
+def ConvertTcpToTosa
```

Comment:
This pass is mainly here to help bootstrap right (and it will go away at some point)?

If yes, please add some commentary here to that effect.

Reply:
That was the initial plan. But after the recent discussions, it felt like this pass might be something the community might find useful. We can probably take a call on this later. I have left a comment mentioning that this will be supported in the longer term if there is a usecase.

Reply:
Makes sense.